Tools/LocBaml/LocBaml.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <Configuration>Release</Configuration>
+    <ProductVersion>8.0.50215</ProductVersion>
```

Comment:
nits

- What do ProductVersion and SchemaVersion do? 
- Should the Configuration be hardcoded to Release? Shouldn't this adapt based on solution-configuration ?

Reply:
These were included in the project in the NETFX repo but are no longer used.  Removed.